src/setIntersect.js

```diff
@@ -0,0 +1,36 @@
+export default function setIntersect( set1 = [], set2 = [] ){
+  if( set1 == null || set1 == null){
```

Comment:
Opps.  That's probably in every set file, too.  We'll fix all four of those in a later commit.